worker.js

```
@@ -1,271 +1,28 @@
+    for (const key in env) {
+      if (
+        typeof key === 'string' &&
+        !['DB', 'ASSETS', 'ENVIRONMENT'].includes(key)
+      ) {
+        process.env[key] = env[key];
       }
-      return new Response(
-        JSON.stringify({ message: 'Habit deleted successfully' }),
-        { headers }
-      );
     }
```

ESLint

[error] for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.

(no-restricted-syntax)